lit/docs/install/web.lit

```diff
@@ -190,6 +181,19 @@ as performing all pipeline scheduling. It's basically the brain of Concourse.
     \code{web} nodes in the cluster to ensure consistent results.
   }
 
+  \section{
+    \title{Connection pools}{web-connection-pooling}
+
+    Furthermore, you may wish to configure the max number of parallel database
+    connections that each node makes. The number you choose should not be
```

Comment:
ooh I don't think this gives the precise meaning of `CONCOURSE_MAX_OPEN_CONNECTIONS` - reading this statement, I'd assume that setting that to 10 would force a `web` node to not go past 10 conns to `db`, but that's not true - the actual max would be `10 * $number_of_pools`

```go
	MaxOpenConnections int `long:"max-conns" description:"The maximum number of open connections for a connection pool." default:"32"`

// ...

	apiConn, err := cmd.constructDBConn(retryingDriverName, logger, cmd.MaxOpenConnections, "api", lockFactory)
	if err != nil {
		return nil, err
	}

	backendConn, err := cmd.constructDBConn(retryingDriverName, logger, cmd.MaxOpenConnections, "backend", lockFactory)
	if err != nil {
		return nil, err
	}

```

I wonder if we should also document that each web node has two pools 🤔 

wdyt?

thx!

Reply:
Should we rename this property entirely (to something like `CONCOURSE_MAX_OPEN_CONNECTIONS_PER_POOL`) or change its behavior to be more in line with the name of the property?

Or is it sufficient to amend our instructions to read:
```
Furthermore, you may wish to configure the max number of parallel database connections per pool that each node makes.
```
...so that it's apparent that this number is per pool instead of a total?

Reply:
In the foreseeable future we will might switch to  having many connection pools (one for scheduling, one for GC, one for API calls, etc) so documenting the specific number of pools will rot -- so maybe we should actually just provide a single knob for the max total number of connections that the web node can consume and dynamically allocate that number across multiple pools. I have heard @vito allude to something like this. The other option is that we should converge on all access to the database being brokered by the API, which basically means only one (API) pool per node, with the scheduler or garbage collector actually making API calls instead of hitting the DB directly. This is an approach I have heard @pivotal-jwinters favour explicitly.